test_serve.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from   serve import app
+
+NEXIST_DANDI_ID = "999999"
+
+@pytest.mark.parametrize('req_url,resp_url', [
+    ("/", "https://gui.dandiarchive.org/"),
+    ("/about", "https://www.dandiarchive.org"),
+    ("/dandiset", "https://gui.dandiarchive.org/#/dandiset"),
+    (
+        "/dandiset/000003",
+        "https://gui.dandiarchive.org/#/dandiset/5e6eb2b776569eb93f451f8d",
+    ),
+    #(
+    #    "/dandiset/000003/???",
+    #    "https://gui.dandiarchive.org/#/dandiset/???",
+    #),
```

Comment:
this would be a version test, which for the moment would return exactly the same response as above.

```suggestion
    (
    "/dandiset/000003/0.20200703.1040",
    "https://gui.dandiarchive.org/#/dandiset/5e6eb2b776569eb93f451f8d",
    ),
```